src/Utility.php

```diff
@@ -46,4 +48,11 @@ public static function prepareParameters(array $parameters)
 
         return preg_replace('/\%5B\d+\%5D/', '%5B%5D', http_build_query($parameters));;
     }
+
+    protected static function needsAmountConversion(array $parameters)
+    {
+        $zeroDecimalCurrencies = [ 'BIF', 'DJF', 'JPY', 'KRW', 'PYG', 'VND', 'XAF', 'XPF', 'CLP', 'GNF', 'KMF', 'MGA', 'RWF', 'VUV', 'XOF' ];
```

Comment:
This list doesn't have `UGX`. https://stripe.com/docs/currencies#zero-decimal

Reply:
You can submit a PR to add it, is a new currency since this was added.